CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 _Summary for the 1.7.1 release: fix a bug when colorizing the log output._
 
 Changelog since 1.7.0:
+### Changed
+ - Changed the default behavior for the default value for `maximum-leds` to be all the LED's instead of one for the Corsair Commander family of devices 
```

Comment:
This entry is the the wrong place: 1.7.1 has already been release.

For this reason and a few others I generally prefer to update the changelog independently of the actual changes, usually once a week.

So you can just drop this and I'll update the changelog later.

P.S. I also considering marking this as a "fix", which I think is reasonable and would also allow us to ship it in a 1.7.2.  This is a bit more relevant than usual because it seems that this release cycle is going slow, so it's possible that the October release may turn out to just be 1.7.2.

Reply:
oops my bad, I just read the since part and put it in, but Ill remove it